docs/application/native/guides/compliations/overview.md

```diff
@@ -0,0 +1,19 @@
+# Watch Face Complications
+
+
+Watch Face complications are features on a watch face which shows specific information in addition to displaying current time.
```

Comment:
Watch Face complications are features on a watch face, which shows specific information in addition to displaying current time.